pkg/branding/brandedtext_test.go

```diff
@@ -0,0 +1,37 @@
+package branding
+
+import (
+	"testing"
+
+	"github.com/stackrox/rox/pkg/testutils/envisolator"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetBrandedProductName(t *testing.T) {
+	envIsolator := envisolator.NewEnvIsolator(t)
```

Comment:
When using `envisolator`, you need to eventually do `envIsolator.RestoreAll()`. The problem is that this is quite often forgotten in our code base. @RTann [pointed](https://srox.slack.com/archives/C02CFR7GQUS/p1649340913298819?thread_ts=1649281444.860079&cid=C02CFR7GQUS) me at https://pkg.go.dev/testing#T.Setenv which should cleanup automatically at the end of the test. Do you want to give it a try?

Reply:
That sounds great, but I'm not sure how to include a new package to our project and make sure CI downloads it too. If it's not too difficult I could look into it, but I'll just call `envIsolator.RestoreAll()` at the end of my tests for now.

Reply:
I think `testing` should be built-in that comes with Go, and it's already imported in this file.

Reply:
No concerns from me against `envisolator`. I just leave this thread open so that if you'd like to use `T.Setenv` some time in the future PRs, you have a reference point.